Description: Fix FTBFS on big endian
Author: Adrian Bunk <bunk@debian.org>
Bug-Debian: https://bugs.debian.org/884202
Applied-Upstream: nightly
--- norm-1.5r6+dfsg1.orig/protolib/include/protoSpace.h
+++ norm-1.5r6+dfsg1/protolib/include/protoSpace.h
@@ -89,7 +89,7 @@ class ProtoSpace
                 }
 #else               
                 ProtoTree::Endian GetEndian() const
-                        return ProtoTree::ENDIAN_BIG;
+                        {return ProtoTree::ENDIAN_BIG;}
                 void SetNode(Node* theNode)
                     {memcpy(key+sizeof(double), &theNode, sizeof(Node*));}
                 Node* GetNode() const
